Makefile

```diff
@@ -1,6 +1,7 @@
 GOCMD=$(or $(shell which go), $(error "Missing dependency - no go in PATH"))
 DOCKER=$(or $(shell which docker), $(error "Missing dependency - no docker in PATH"))
 GOBINPATH=$(shell $(GOCMD) env GOPATH)/bin
+PROTOC=$(or $(shell which protoc), $(error "Missing protobuf compilter - no protoc on PATH"))
```

Comment:
Shouldn't all of these be `:=` and not `=`?  We don't need to run a shell to find which program to run before _each_ time we use one of these macros??!?

Reply:
good point - I'll update